model/src/main/java/com/javahelp/model/user/Gender.java

```diff
@@ -0,0 +1,7 @@
+package com.javahelp.model.user;
+
+public enum Gender {
+    MALE,
+    FEMALE,
+    OTHERS
```

Comment:
just a thought, but would it be possible to make it so that users can specify their gender if they pick other?

Reply:
I know there are a bunch of other groups too, but how do you all think? Is `OTHERS` needed to be more specific?

Reply:
> just a thought, but would it be possible to make it so that users can specify their gender if they pick other?

Ye that's a really great argument for making it an unrestricted string

Reply:
Maybe, we can do something like male, female, and if other, please specify...
In the UI, it would be buttons and if they pick other, then a text would jump up and they can type their genders

Reply:
I discussed with Professor Jonathan, and he said when `OTHERS` is selected, there will be another option for them to enter a costume string that represents their own gender.